cmake/third_party.cmake

```
@@ -236,6 +236,33 @@ if(THIRD_PARTY)
         EXCLUDE_FROM_ALL)
     endforeach()
   endif(BUILD_PYTHON)
+  install(
+    DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/third_party_install/llvm/include/
```

这里是不是应该用 THIRD_PARTY_DIR
```suggestion
    DIRECTORY ${THIRD_PARTY_DIR}/llvm/include/
```

---

llvm是在oneflow/ir目录下的，不是THIRD_PARTY_DIR，但它会被install到third_party_install/llvm/目录下。不过有一个问题是如果LLVM_PROVIDER是in-tree的话，还会install么 @jackalcooper